fileutil/perm_other.go

```diff
@@ -0,0 +1,43 @@
+// +build !windows
+
+package fileutil
+
+import (
+	"os"
+	"os/user"
+	"strconv"
+	"syscall"
+)
+
+func HasPermissionToDir(info os.FileInfo) (bool, error) {
```

Comment:
given how we only use the bool here, have you thought about omitting either of the return values?